tests/functional/firmware/k21-rtt/asm/bootloader.s

```diff
@@ -0,0 +1,188 @@
+/* File: bootloader.s
```

Comment:
Hah! I wasn't expecting for this to ever see the light of day. I guess what goes around comes around.